app-catalog/src/settings.tsx

```diff
@@ -0,0 +1,106 @@
+import { HoverInfoLabel, NameValueTable } from '@kinvolk/headlamp-plugin/lib/components/common';
+import { Box, Switch, SwitchProps, Typography, useTheme } from '@mui/material';
+import { useState } from 'react';
+import { store } from './components/charts/List';
+
+export function AppCatalogSettings() {
+  const config = store.get();
```

Comment:
same here, use `store.useConfig()`

Reply:
using the store.useConfig() here seems to not work here also

Reply:
had to rework it a little